examples/04.Audio_Input/Audio_Input/Audio_Input.ino

```diff
@@ -1,11 +1,10 @@
 /*  Test of audio input using Mozzi sonification library.
 
- An audio input using the range between 0 to 5V on analog pin A0
- is sampled and output on digital pin 9.
+ An audio input using the range between 0 to 5V on analog pin A0 (or as
```

Comment:
Detail, but now there is quite some support for 3.3V boards, I would just change that to make the 5V less absolute (not related to the PR)

Reply:
True. There are more facets to the problem, too. E.g. many examples give a specific pinout / circuit for Arduino Uno. Helpful as that is, it is no longer accurate in many cases. Not sure, what could be a clean and helpful way out, so I ignored that for now. Adding as a task item to #210 for the time being.